app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest.kt

```diff
@@ -893,15 +896,42 @@ class StateFragmentTest {
   }
 
   @Test
-  fun testStateFragment_inputRatio_submit_correctAnswerDisplayed() {
+  fun testStateFragment_inputRatio_correctAnswerSubmitted_correctAnswerDisplayed() {
     launchForExploration(TEST_EXPLORATION_ID_6).use {
       startPlayingExploration()
       typeRatioExpression("4:5")
 
       clickSubmitAnswerButton()
 
       onView(withId(R.id.submitted_answer_text_view))
-        .check(matches(ViewMatchers.withContentDescription("4 to 5")))
+        .check(matches(withContentDescription("4 to 5")))
+    }
+  }
+
+  @Test
+  @RunOn(TestPlatform.ROBOLECTRIC)
+  fun testStateFragment_inputRatio_correctAnswerSubmitted_correctTextBannerDisplayed() {
+    launchForExploration(TEST_EXPLORATION_ID_6).use {
+      startPlayingExploration()
+      typeRatioExpression("4:5")
+
+      clickSubmitAnswerButton()
+
+      onView(withId(R.id.congratulations_text_view))
+        .check(matches(withEffectiveVisibility(VISIBLE)))
+    }
+  }
+
+  @Test
+  @RunOn(TestPlatform.ROBOLECTRIC)
+  fun testStateFragment_inputRatio_correctAnswerSubmitted_confettiDisplayed() {
```

Comment:
can we have a similar test case for landscape? 

`testStateFragment_configChange_inputRatio_correctAnswerSubmitted_confettiDisplayed`

Reply:
Done

Removed this test specifically in preference of the LocalTests but have added landscape cases to them